src/Forerkort/Seksjon.elm

```diff
@@ -0,0 +1,1117 @@
+                    , lagtTilSpørsmålCmd model.debugStatus
+                    )
+                        |> IkkeFerdig
+
+                _ ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+        SendSkjemaPåNytt ->
+            case model.aktivSamtale of
+                LagringFeilet error skjema ->
+                    ( error
+                        |> LagreStatus.fraError
+                        |> LagrerFørerkort skjema
+                        |> nesteSamtaleSteg model (Melding.svar [ "Ja, prøv på nytt" ])
+                    , Api.postFørerkort FørerkortLagret skjema
+                    )
+                        |> IkkeFerdig
+
+                _ ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+        WindowEndrerVisibility visibility ->
+            case visibility of
+                Visible ->
+                    case model.aktivSamtale of
+                        LagrerFørerkort skjema lagreStatus ->
+                            ( lagreStatus
+                                |> LagreStatus.setForsøkPåNytt
+                                |> LagrerFørerkort skjema
+                                |> oppdaterSamtaleSteg model
+                            , Cmd.none
+                            )
+                                |> IkkeFerdig
+
+                        LagringFeilet error skjema ->
+                            if ErrorHåndtering.operasjonEtterError error == LoggInn then
+                                IkkeFerdig
+                                    ( error
+                                        |> LagreStatus.fraError
+                                        |> LagrerFørerkort skjema
+                                        |> oppdaterSamtaleSteg model
+                                    , Api.postFørerkort FørerkortLagret skjema
+                                    )
+
+                            else
+                                IkkeFerdig ( Model model, Cmd.none )
+
+                        _ ->
+                            IkkeFerdig ( Model model, Cmd.none )
+
+                Hidden ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+
+validertSkjemaTilSetninger : ValidertFørerkortSkjema -> List String
+validertSkjemaTilSetninger validertSkjema =
+    [ "Førerkort: " ++ Skjema.førerkortFraValidertSkjema validertSkjema
+    , "Fra dato: " ++ Skjema.fraDatoFraValidertSkjema validertSkjema
+    , "Utløper dato: " ++ Skjema.tilDatoFraValidertSkjema validertSkjema
+    ]
+
+
+updateEtterAtBrukerHarValgtFørerkortFraDropdown : Samtale -> List FørerkortKode -> String -> Samtale
+updateEtterAtBrukerHarValgtFørerkortFraDropdown aktivSamtale remoteFørerkortKoder valgtFørerkort =
+    case ( aktivSamtale, remoteFørerkortKoder ) of
+        ( VelgNyttFørerkort velgNyttFørerkortInfo, førerkortKoder ) ->
+            case List.find (\forerkortKode -> valgtFørerkort == FørerkortKode.kode forerkortKode) førerkortKoder of
```

Comment:
Har du ikke en funksjon for dette? Den jeg sa du kunne rename til FørerkortKode.fraString?